config/navigation.txt

```diff
@@ -54,6 +54,7 @@ Learn
   DEPLOY
     [/learn/deploy/deployment]
     Release strategy
```

Comment:
We probably need to re-think this section a bit, since its probably not very clear what is different between policy and strategy :P but this can be done later.

Reply:
Agreed! A lot of the nav could do with an overhaul particularly as I'm starting to add additional content in addition to refreshing outdated stuff. I'll create a new issue to track this as it'll be a useful exercise to identify missing content.